test/unit/specs/utils/assignIf.spec.js

```diff
@@ -0,0 +1,14 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+// eslint-disable-next-line no-unused-vars
+import assignIf from "../../../../src/utils/assignIf";
```

Comment:
I'd like to know how code was pushed without this spec file created.

Reply:
I think this is on me @Aaronius, I'll double check why the script is not running on my machine.